data-serving/scripts/convert-data/constants.py

```diff
@@ -54,18 +54,24 @@
     ('bergamo', 'lombardia', 'italy'): ('Bergamo', 'Lombardia', 'Italy'),
     ('harbin',): ('Harbin City', 'Heilongjiang', 'China'),
     ('lombardy', 'italy'): (None, 'Lombardia', 'Italy'),
-    ('lombardy italy',): (None, 'Lombardia', 'Italy'),
     ('london',): ('London', 'England', 'United Kingdom'),
     ('madrid',): (None, 'Madrid', 'Spain'),
+    ('madrid', 'spain'): (None, 'Madrid', 'Spain'),
+    ('miami', 'florida', 'united states'): ('Miami-Dade County', 'Florida', 'United States'),
```

Comment:
What does this data represent? The comment says `city, province, country` -- but we're using Miama-Dade County in lieu of Miami? And, totally without context, I'd guess that `miami` is most often a reference to the city as opposed to the county.

Reply:
that's fair -- then miami the city needs to be added to the geo tsv. I'll do that instead